text/0000-layer-origin-metadata.md

```diff
@@ -0,0 +1,234 @@
+# Meta
+[meta]: #meta
+- Name: Layer Origin Metadata
+- Start Date: 2020-07-09
+- Author(s): pclalv
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: (put "N/A" unless this replaces an existing RFC, then link to that RFC)
+
+# Summary
+[summary]: #summary
+
+<!-- One paragraph explanation of the feature. -->
+
+Exposing layer origin metadata in `report.toml` will improve layer
```

Comment:
Maybe we could use the term "traceability" instead of "reproducibility" throughout this document to avoid potential confusion with the other (related but subtly distinct) meaning of "build reproducibility"? https://github.com/buildpacks/spec/pull/111

Reply:
I would agree with your distinction @pclalv.